README.md

```diff
@@ -4,8 +4,10 @@ Quiver
 Quiver is a set of utility libraries for Dart that makes using many Dart
 libraries easier and more convenient, or adds additional functionality.
 
-[![Build Status](https://travis-ci.org/google/quiver-dart.svg?branch=master)](https://travis-ci.org/google/quiver-dart)
+[![Build Status](https://github.com/google/quiver-dart/workflows/Dart%20CI/badge.svg)](https://github.com/google/quiver-dart/actions?query=workflow%3A"Dart+CI"+branch%3Amaster)
```

Comment:
Is "Dart CI" the right label for it?

Reply:
I'm not necessarily the biggest fan of it, but it's the name of the GitHub Actions workflow and what our other repos use. e.g. 
https://github.com/dart-lang/markdown
https://github.com/dart-lang/source_maps